app/decorators/decidim/initiatives/initiatives_controller_decorator.rb

```diff
@@ -11,7 +11,7 @@
   # Assigns the value "closed" to the :state key until the given date is reached.
   def default_filter_params
     original_default_filter_params.tap do |default_params|
-      default_params[:state] = "closed" if Time.current < SIGNATURE_START_DATE
+      default_params[:state] = Time.current < SIGNATURE_START_DATE ? "open" : "closed"
```

Comment:
Shouldn't it be the oher way around?
```suggestion
      default_params[:state] = Time.current < SIGNATURE_START_DATE ? "closed" : "open"
```

Reply:
**No, in production the default state right now is "open" and they want "closed" to be shown instead.**
![imagen](https://user-images.githubusercontent.com/56717126/75959236-1c4fa400-5ebe-11ea-89c6-36d6a9e74537.png)